### PR TITLE
Revert "Remove KUBERNETES_VERSION var"

### PR DESCRIPTION
**Pull request by fmuyassarov**
This reverts https://github.com/metal3-io/project-infra/pull/232. Apparently, we were wrong to expose
KUBERNETES_VERSION to JJB, since it complicates
the var tuning in the future, and now we have too many
copies of the var in JJB files.

### CODE REVIEW

File: jenkins/scripts/files/run_integration_tests.sh [hunk 1]
```diff
@@ -12,14 +12,18 @@ export CAPM3_VERSION
...
 
 if [ "${REPO_NAME}" == "airship-dev-tools" ]
 then
   export IMAGE_NAME
   export IMAGE_LOCATION
 fi
 
+if [ "${CAPM3_VERSION}" == "v1alpha4" ]
+then
+  export KUBERNETES_VERSION="v1.21.2"
```
Comment (Sunnatillo) [inline jenkins/scripts/files/run_integration_tests.sh:24]: are not we using "v1.22.0" KUBERNETES_VERSION?

Comment (fmuyassarov) [reply jenkins/scripts/files/run_integration_tests.sh:24]: for v1a4 no

**Review by kashifest — APPROVED** [review]

**Review by smoshiur1237 — APPROVED** [review]

**Review by smoshiur1237 — COMMENTED** [review]
/lgtm

**Comment by furkatgofurov7** [comment]
@fmuyassarov 
Does this need [cicd/10629](https://gerrit.nordix.org/c/infra/cicd/+/10629) before merging this patch?

**Comment by fmuyassarov** [comment]
> @fmuyassarov Does this need [cicd/10629](https://gerrit.nordix.org/c/infra/cicd/+/10629) before merging this patch?

yep, we merged it, and that's why I re-triggered v1a4 after you. So, hopefully will go smooth

**Comment by kashifest** [comment]
/lgtm
This is causing CI failure

**Pull request merged**